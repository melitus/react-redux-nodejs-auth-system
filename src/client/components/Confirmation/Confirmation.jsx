import React from 'react';


const Confirmation = () => (
  <div>
    Registration successful
  </div>
);

export default Confirmation;
