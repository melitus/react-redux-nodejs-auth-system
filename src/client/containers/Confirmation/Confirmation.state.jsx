import { connect } from 'react-redux';
import Confirmation from '../../components/Confirmation/Confirmation';

export const mapDispatchToProps = {};

export const mapStateToProps = () => ({});

export default connect(mapStateToProps, mapDispatchToProps)(Confirmation);
