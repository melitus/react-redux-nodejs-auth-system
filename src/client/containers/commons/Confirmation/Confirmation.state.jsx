import { connect } from 'react-redux';
import Confirmation from '../../pages/Confirmation/Confirmation';

export const mapDispatchToProps = {};

export const mapStateToProps = () => ({});

export default connect(mapStateToProps, mapDispatchToProps)(Confirmation);
