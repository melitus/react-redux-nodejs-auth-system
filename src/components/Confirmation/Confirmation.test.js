import React from 'react';
import enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-15.4';

import Confirmation from './Confirmation';

enzyme.configure({ adapter: new Adapter() });

describe('<Confirmation />', () => {
  it('should render default snapshot', () => {
    const wrapper = shallow(
      <Confirmation />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
