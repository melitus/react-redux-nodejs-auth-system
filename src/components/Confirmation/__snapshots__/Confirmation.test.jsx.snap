exports[`<Confirmation /> should render default snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Confirmation />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "Registration successful",
    },
    "ref": null,
    "rendered": "Registration successful",
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Registration successful",
      },
      "ref": null,
      "rendered": "Registration successful",
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFifteenFourAdapter {
      "options": Object {
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
